crates/web5/src/dids/document.rs

```diff
@@ -10,7 +10,7 @@ pub struct Document {
     pub controller: Option<Vec<String>>,
     #[serde(rename = "alsoKnownAs", skip_serializing_if = "Option::is_none")]
     pub also_known_as: Option<Vec<String>>,
-    #[serde(rename = "verificationMethod")]
+    #[serde(rename = "verificationMethod", default = "Vec::new")]
```

Comment:
Clever. This may have implications but I'm fine with moving forward with it.

Reply:
I added it because the example `did:web` (https://tbd.website/.well-known/did.json) was failing to Deserialize, complaining about `verification_methods` being missing -- though maybe there's another way (aside from #233)?

Reply:
We should update that DID because it's not really adding any value. I'm not sure how it can be used in its current state.